internal/module/openapi.go

```diff
@@ -214,7 +213,7 @@ func parseString(key, pattern string, result map[string]any) error {
 		}
 		result[key] = r
 	} else {
-		result[key] = base64.StdEncoding.EncodeToString([]byte(key))
+		result[key] = key
```

Comment:
since we now use our function for base64, we don't need to further process string values.